code/modules/surgery/surgery.dm

```diff
@@ -513,3 +523,26 @@
 		for(var/reagent in chems_needed)
 			if(target.reagents.has_reagent(reagent))
 				return TRUE
+
+/datum/surgery_step/proc/play_preop_sound(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	if(!preop_sound)
+		return
+	var/sound_file_use
+	if(islist(preop_sound))
```

Comment:
Why only have this logic for the preop sound? Why not extend it to the success/fail sound, too?

Reply:
Only needed a list for preop sounds, that's how TG did it and it'd be unused code if I extended it to the others.

Reply:
Fair enough, guess it could be expanded later if we decide we want it. Don't worry about it for now